analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ControllingPermissions.cs

```diff
@@ -42,7 +50,7 @@ void CreatingPermissions()
         {
             WindowsIdentity.GetCurrent(); // Noncompliant
 //          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+            
```

Comment:
pedantic: white space can be removed.

PS: so strange to write this comment while using a dark theme 😄

Reply:
IDK what whitespace you are talking about, on git it is greenspace 😄